cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -65,28 +66,34 @@ Instead they are defined in terms of their underlying _privileges_ and they can
 In addition any new roles created can by assigned any combination of _privileges_ to create the specific access control desired.
 A major additional capability is _sub-graph_ access control whereby read-access to the graph can be limited to specific combinations of label, relationship-type and property.
 
+
 [[administration-security-syntax-rules]]
 == Syntax summaries
-Almost all administration commands have variations in the commands. Parts of the command that are optional or can have multiple values are most common.
-To show all versions of a command, a summary of the syntax will be presented. These summaries will use some special characters to indicate such variations.
+
+Almost all administration commands have variations in the commands.
+Parts of the command that are optional or can have multiple values are most common.
+To show all versions of a command, a summary of the syntax will be presented.
+These summaries will use some special characters to indicate such variations.
 
 The special characters and their meaning are as follows:
 
 .Special characters in syntax summeries
-[options="header", width="100%", cols="1a,3,3"]
+[options="header", width="100%", cols="1a^,3a,3a"]
```

Comment:
Yeah I have no real clue what these numbers and letters mean :P

Reply:
😄  this is asciidoc syntax for the table layout - nothing that we'd expect you to add to your PRs, and is something we're happy to pick up in reviews.

The `1a^` means column width of 1, and that content within is asciidoc (`a`)markup, and to center the text (`^).
The `3a` means the columns should have width of 3, and also have asciidoc markup. 

Here is a useful guide: https://blog.mrhaki.com/2014/11/awesome-asciidoctor-styling-columns-and.html